tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md

```diff
@@ -1,8 +1,8 @@
 ---
 author_name: Gergana Tsakova
 author_profile: https://github.com/Joysie
-title: Create a Python Application via Cloud Foundry CLI
-description: Create a simple Python application in CF CLI and enable services for it.
+title: Create a Python Application via CF CLI
+description: Create a simple Python application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
```

Comment:
After this step you can use the abbreviation "CF CLI". Please update the tutorial accordingly.